packages/fx-core/src/component/driver/deploy/azure/azureFunctionDeployDriver.ts

```diff
@@ -27,6 +27,13 @@ export class AzureFunctionDeployDriver implements StepDriver {
       context.logProvider
     );
   }
+
+  execute(args: unknown, ctx: DriverContext): Promise<ExecutionResult> {
+    return wrapSummary(this.run.bind(this, args, ctx), [
+      // eslint-disable-next-line no-secrets/no-secrets
```

Comment:
Just curious on why we need this comment to suppress only in this azureFunctionsDriver